beer_and_tell/2016-09-September.md

```diff
@@ -0,0 +1,80 @@
+Once a month, web developers from across the Mozilla Project get together to talk about our side projects and drink, an occurrence we like to call "Beer and Tell".
+
+There's a [wiki page available][wiki] with a list of the presenters, as well as links to their presentation materials. There's also a [recording available][recording] courtesy of Air Mozilla.
+
+[wiki]: https://wiki.mozilla.org/Webdev/Beer_And_Tell/2016/September
+[recording]: https://air.mozilla.org/webdev-beer-and-tell-september-2016/
+
+## emceeaich: Gopher Tessel
+First up was [emceeaich][], who shared [Gopher Tessel][], a project for running a [Gopher][] server on a [Tessel][]. Tessel is small circuit board that runs [Node.js][] projects; Gopher Tessel reads sensors (such as the temperature sensor) connected to the board, and exposes their values via Gopher. It also can control lights connected to the board.
```

Comment:
There should be some context about Gopher protocol, such as "running a Gopher server, which was an Internet protocol popular before the World Wide Web, and now largely forgotten".